### DISCUSSION TITLE
Storing Active Websocket Connections in Production

### DISCUSSION THREAD

**Discussion by BenA-SA** [cookiecutter/cookiecutter-django#4946] [Q&A]
Hi,

I am trying to store a list of active websocket connections so that I can send them all a message when a model object is created. In development, this works fine - I have an empty list at the top of my websocket.py file, when a websocket.connect event occurs, I append the send to the list, and then remove it when websocket.disconnect occurs.

However this list does not seem to store properly in production (from google searching, this is due to multiple processes I think). I looked at trying to cache the send event, or save it to a model, but I get a pickl error when trying to do that.

Is there a "best way" to achieve this within the cookiecutter-django framework?

Thanks for any help anyone can offer!

**Comment by foarsitter** [comment]
Never used it before but I think that https://github.com/encode/broadcaster solves your problem.

**Reply by BenA-SA** [reply]
Thanks for the suggestion, struggling to get it working but I'll keep trying and let you know if I can get it working.
